### PR TITLE
Added Papermc Server

### PR DESCRIPTION
**Pull request by OleHild**

### CODE REVIEW

File: manifests/papermc/deployments.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            - name: minecraft
+              containerPort: 25565
+              protocol: TCP
+          volumeMounts:
+            - name: datadir
+              mountPath: /data
+      volumes:
+        - name: datadir
+          persistentVolumeClaim:
+            claimName: minecraft-server-creative-datadir
```
Comment (durandom) [inline manifests/papermc/deployments.yaml]: change to `minecraft-server-paper-datadir`

File: manifests/papermc/persistent-volume-claims.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: minecraft-server-creative-datadir
```
Comment (durandom) [inline manifests/papermc/persistent-volume-claims.yaml]: change to `minecraft-server-paper-datadir`

**Review by durandom — CHANGES_REQUESTED** [review]

**Comment by goern** [comment]
Was macht der Server?

**Pull request merged**